torch/_higher_order_ops/effects.py

```diff
@@ -210,6 +212,8 @@ def _get_schema(op, args) -> torch.FunctionSchema:
         return op._schema
     elif op == call_torchbind:
         return getattr(args[0], args[1]).schema
+    elif op == hop_print:
+        return op.gen_schema(args)
```

Comment:
do we need to update _get_schema to take in kwargs?

Reply:
I think we should since the key might be kwargs that we want to wrap for effect prevention?